unit/torque.go

```diff
@@ -0,0 +1,94 @@
+// Code generated by "go generate gonum.org/v1/gonum/unit”; DO NOT EDIT.
+
+// Copyright ©2014 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package unit
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"unicode/utf8"
+)
+
+// Torque represents a torque in Newton meters.
+type Torque float64
+
+const (
+	Yottanewtonmeter Torque = 1e24
+	Zettanewtonmeter Torque = 1e21
+	Exanewtonmeter   Torque = 1e18
+	Petanewtonmeter  Torque = 1e15
+	Teranewtonmeter  Torque = 1e12
+	Giganewtonmeter  Torque = 1e9
+	Meganewtonmeter  Torque = 1e6
+	Kilonewtonmeter  Torque = 1e3
+	Hectonewtonmeter Torque = 1e2
+	Decanewtonmeter  Torque = 1e1
+	Newtonmeter      Torque = 1.0
+	Decinewtonmeter  Torque = 1e-1
+	Centinewtonmeter Torque = 1e-2
+	Millinewtonmeter Torque = 1e-3
+	Micronewtonmeter Torque = 1e-6
+	Nanonewtonmeter  Torque = 1e-9
+	Piconewtonmeter  Torque = 1e-12
+	Femtonewtonmeter Torque = 1e-15
+	Attonewtonmeter  Torque = 1e-18
+	Zeptonewtonmeter Torque = 1e-21
+	Yoctonewtonmeter Torque = 1e-24
+)
+
+// Unit converts the Torque to a *Unit
+func (t Torque) Unit() *Unit {
+	return New(float64(t), Dimensions{
+		LengthDim: 2,
+		MassDim:   1,
+		TimeDim:   -2,
+	})
+}
+
+// Torque allows Torque to implement a Torqueer interface
```

Comment:
This isn't the doc we want, but this fix can happen in a different PR (it may be elsewhere, I haven't checked).

Reply:
Fixed.